server/src/main/java/org/elasticsearch/health/HealthPeriodicLogger.java

```diff
@@ -204,7 +204,10 @@ private HealthPeriodicLogger(
         this.logWriter = logWriter == null ? logger::info : logWriter;
 
         // create metric for overall level metrics
-        this.redMetrics.put("overall", LongGaugeMetric.create(this.meterRegistry, "es.health.overall.red", "Overall: Red", "{cluster}"));
+        this.redMetrics.put(
+            "overall",
+            LongGaugeMetric.create(this.meterRegistry, "es.health.overall.red.status", "Overall: Red", "{cluster}")
```

Comment:
Trying to understand "status" here: why not "count" (0/1)?

Reply:
this is a workaround for a lack of enum/boolean support in OTEL. 
count would indicate that this value will be increasing.